source/documentation/test-scenarios.html.md.erb

```diff
@@ -330,24 +330,22 @@ A 503 Service Unavailable error response will take the form of
 
 ## Endpoint: Get all messages for an excise movement using the localReferenceNumber
 
-You must make sure you have first tested the [Submit a draft excise movement (IE815) endpoint](#endpoint-submit-a-new-draft-excise-movement). You must have used one of the LRNs below in the body of the XML message, and received a success response.
+You must make sure you have first tested the [Submit a draft excise movement (IE815) endpoint](#endpoint-submit-a-new-draft-excise-movement) for each lrn in the below scenario. You must have used one of the LRNs below in the body of the XML message, and received a success response.
```

Comment:
Chris or Nicole can you double check this? I have added a little text "for each lrn in the below scenario" to highlite that the user need to run the submit draft movement for each lrn before calling the Get Messages otherwise it will get an error.

Reply:
Looks good just need to capitalise LRN like it is in the next sentence I'd say.